test/models/Reads/PensionSchemeAdministratorReadsCompanySpec.scala

```diff
@@ -34,7 +33,8 @@ class PensionSchemeAdministratorReadsSpec extends WordSpec with MustMatchers wit
         "customerType" -> "TestCustomer",
         "idType" -> JsString("TestId"),
         "idNumber" -> JsString("TestIdNumber")),
-        "contactDetails" -> Json.obj("phone" -> "07592113", "email" -> "test@test.com"),
+        "phone" -> JsString("07592113"),
```

Comment:
have you moved this out of contact details wrapper?

Reply:
No, moved back in